tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -12,6 +12,48 @@ let timeOfLastDeleteOperation = Date.now()
 let timeOfLastUploadOperation = Date.now()
 const { download } = require('../helpers/webdavHelper')
 
+const getAllFolders = async function () {
+  const filenames = await client.page.filesPage().getAllFilesFolders()
+  const folders = filenames.filter(elements => elements.endsWith('/'))
```

Comment:
can be an extra function in webdavhelper `getAllFolders` that function would use `getAllFilesFolders` and do the filtering

Reply:
fixed